src/backend/services/BackendAnalysisService.ts

```
@@ -0,0 +1,261 @@
+      console.error(`Gemini API call failed for ${sectionName}:`, error);
+      throw new Error(`Analysis failed for ${sectionName}: ${error instanceof Error ? error.message : 'Unknown error'}`);
+    }
+  }
+
+  /**
+   * Get appropriate prompt for analysis section
+   */
+  private getPromptForSection(sectionKey: string, sectionName: string, scriptText: string): string {
+    // Check if we have a custom prompt for this section
+    const customPrompt = this.promptConfig[sectionKey];
+    if (customPrompt && customPrompt.prompt) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)